core/drivers/crypto/se050/session.c

```diff
@@ -39,10 +39,16 @@ TEE_Result se050_core_early_init(struct se050_scp_key *keys)
 	return TEE_SUCCESS;
 }
 
-static TEE_Result display_info(void)
+static TEE_Result update_se_info(void)
 {
-	se050_display_board_info(se050_session);
-	/* the session must be closed after accessing board information */
+	sss_status_t status = kStatus_SSS_Success;
+
+	status = se050_get_se_info(se050_session,
+				   IS_ENABLED(CFG_CORE_SE05X_DISPLAY_INFO));
+	if (status != kStatus_SSS_Success)
+		return TEE_ERROR_GENERIC;
+
+	/* the session must be closed after accessing the board information */
```

Comment:
if so, should this close the session before returning an error at line 49?

Reply:
ok. althought it doesnt really matter at this point. the device cant be recovered....will have to reboot.